downstream/modules/platform/proc-controller-importing-subscriptions.adoc

```diff
@@ -45,7 +45,7 @@ This option is checked by default, but you can opt out of any of the following:
 * *Insights Analytics*. Provides a high level analysis of your automation with {ControllerName}. 
```

Comment:
Subscription info is now in the Access management guide so I think this module can be removed.

Reply:
I commented out the include for this file but also updated it, in case it accidentally gets used somewhere else.